flow-model-generator/modules/flow-model-generator-ls-extension/src/main/resources/copilot/instructions/ballerina/http/test.md

```
@@ -0,0 +1,43 @@
+- Start with necessary imports, including `ballerina/http`, `ballerina/test` any other imports that is required.
+- Define an HTTP Client at the module level named `clientEp`
+- Organize tests logically: create, read, update, delete
+- Create helper functions only when improving readability
+- Don't redefine existing types from the codebase, reuse them.
+
+## Test Function Generation:
+- MUST generate AT LEAST ONE test case for EACH resource function
+- Ensure each test function returns `error?` and use `check` keyword for error propagation.
+- Use proper HTTP method invocation syntax as per the provided example.
+    - GET:
+        - Ballerina Resource -> resource function get books/[string isbn]() returns Book|Error
```

markdownlint-cli2 (0.20.0)

[warning] No empty links

(MD042, no-empty-links)